### DISCUSSION TITLE
problem finding own module

### DISCUSSION THREAD

**Discussion by luhborba** [mkdocstrings/mkdocstrings#654] [Q&A] [answered]
I'm trying to build documentation using MkDocs and using Mkdocstring-python, to make this documentation more agile, but when I call a module created in the project it doesn't find it.

Basically it returns this error:

ERROR - mkdocstrings: etl.extract.download_kaggle_dataset could not be found
ERROR - Error reading page 'index.md':
ERROR - Could not collect 'etl.extract.download_kaggle_dataset'

![Screenshot_38](https://github.com/mkdocstrings/mkdocstrings/assets/6145108/4515a0b4-bccc-48a3-b368-3b08ae1bfd65)

The folder structure is like this, the file I want to access is inside the etl folder:

![image](https://github.com/mkdocstrings/mkdocstrings/assets/6145108/4fc93c64-3004-4d2f-84e6-95c5cd83b9c9)

however, when I call a function that is inside any of the files present in the etl folder, or any other folder, it returns the aforementioned error:

my mkdocs.yml:

![image](https://github.com/mkdocstrings/mkdocstrings/assets/6145108/0a8d3ec6-80b3-460f-85fe-ace68f0697f8)

my arquivo.md I'm using mkdocstring

![image](https://github.com/mkdocstrings/mkdocstrings/assets/6145108/aa165da5-e236-4d18-b60e-a9a92f1997d6)

my extract.py
![image](https://github.com/mkdocstrings/mkdocstrings/assets/6145108/2d996b25-72a0-4dbb-b8f5-a06b48345883)

I have already accessed several possible solutions but I was not successful in any of them.

Thanks

**Comment by pawamoy** [comment]
Hi, can you run MkDocs again but in verbose mode, and paste the logs here (no screenshot)? `mkdocs build -v`.

**Comment by luhborba** [comment]
the execute `mkdocs serve -v` this is result:

![image](https://github.com/mkdocstrings/mkdocstrings/assets/6145108/6192776b-d525-4bad-89b8-bf90f217aa5e)

![image](https://github.com/mkdocstrings/mkdocstrings/assets/6145108/fb7f4604-cfc9-4831-af13-9b84efb52fd7)

**Comment by pawamoy** [comment]
I explicitly asked for no screenshots...
Please share a link to your repository.
Moving this to a discussion!

**Comment by luhborba** [comment]
Sorry man, 
I don't speak English very well, I did a quick read and didn't see the 'no'.

[https://github.com/luhborba/dataglowup30](https://github.com/luhborba/dataglowup30)

**Reply by pawamoy** [reply]
That's alright, thanks for the link, I'll try your setup.

**Comment by pawamoy** [comment]
OK so that's a regression, probably since we started making search paths absolute for other reasons. Will push a fix and a release today :slightly_smiling_face:

**Reply by pawamoy** [reply, marked as answer]
Should be fixed in v0.41.2, let me know if you have any other issue!
And thanks for reporting :slightly_smiling_face:

**Reply by luhborba** [reply]
Thanks men 😄
